modules/javafx.graphics/src/main/java/com/sun/prism/GraphicsPipeline.java

```diff
@@ -235,8 +235,13 @@ public static GraphicsPipeline createPipeline() {
                     newPipeline = null;
                 }
                 if (PrismSettings.verbose) {
-                    System.err.println("GraphicsPipeline.createPipeline: error"+
+                    System.err.println("GraphicsPipeline.createPipeline: error" +
                                        " initializing pipeline "+ className);
```

Comment:
Since you fixed one minor preexisting formatting issue with a missing space before the `+`, do you want to fix the one on the next line (239), too?